scene/gui/item_list.h

```diff
@@ -143,6 +143,14 @@ class ItemList : public Control {
 
 	bool do_autoscroll_to_bottom = false;
 
+	void _scroll_changed(double);
+	void _shape_text(int p_idx);
+	void _mouse_exited();
+	void _shift_range_select(int p_from, int p_to);
+
+	String _atr(int p_idx, const String &p_text) const;
+
+protected:
```

Comment:
Making `ThemeCache` protected makes it impossible for inheriting classes to implement their own. Though I see why is it needed here; I guess it's something to worry about once it actually becomes a problem (unlikely).